Makefile

```diff
@@ -25,12 +25,22 @@ ENVIRONMENT_FILE := pdk_regression.yml
 # Path to regression
 KLAYOUT_TESTS := klayout/drc/testing/
 
+
 include third_party/make-env/conda.mk
 
 # Lint python code
-lint: | $(CONDA_ENV_PYTHON)
+lint_python: | $(CONDA_ENV_PYTHON)
 	@$(IN_CONDA_ENV) flake8 .
 
+# Lint ruby code
+lint_ruby:| $(CONDA_ENV_PYTHON)
+	@$(IN_CONDA_ENV) gem install rubocop
+	@$(IN_CONDA_ENV) rubocop=`which rubocop`
+	@$(IN_CONDA_ENV) rubocop_dir=`dirname $$rubocop`
+	@$(IN_CONDA_ENV) echo $$rubocop_dir
+	@$(IN_CONDA_ENV) ln -s ../../../bin/ruby $$rubocop_dir/ruby
```

Comment:
that shouldn't be necessary if you install ruby in the env.

Reply:
I have added ruby as a dependency in the `pdk_regression.yml` file and it still causes an issue.

![image](https://user-images.githubusercontent.com/91015308/216909864-650ef6fa-4cb7-4d72-872a-703851d0d57d.png)

Reply:
by default `ruby` shows up `$CONDA_PREFIX/bin` for me when installing it with conda, can you clarify:
- which version of ruby got installed
- which version of rubygem got installer
- how you are installing rubocop

maybe this could be an acceptable workaround:
```
ln -s $CONDA_PREFIX/bin/ruby $CONDA_PREFIX/share/rubygems/bin/ruby
```
wdyt?

Reply:
I have added ruby in yml file as shown:

```yaml
name: pdk_regression
channels:
  - litex-hub
  - defaults
dependencies:
  - klayout
  - python
  - ruby
  - pip
  - pip:
    - -r requirements.txt
```
I have installed rubocop using `gem install rubocop`

The solution you have proposed above is exactly the same as the one existed before.

Please advise how to move forward.

Reply:
@proppy 

Please see below the versions of ruby and gem in the conda environment 

```
Ruby: ruby 3.1.2p20 (2022-04-12 revision 4491bb740a) [x86_64-linux]
Gem: 3.3.7
```

Reply:
I'd prefer for is to use $CONDA_PREFIX to create the link and to track the ruby dependencies using `Gemfile`